src/lib/response-triggers/finish.ts

```diff
@@ -38,7 +40,7 @@ export default class Finish extends Trigger {
 		return pattern.test(message.html);
 	}
 
-	async createMessage() : Promise<MessageObject> {
+	async createMessage(commandMessage :MessageObject, room : string) : Promise<MessageObject> {
```

Comment:
` : `

Reply:
Also, is `commandMessage` being used anywhere? I can't see it being referenced.

Reply:
its not being used, but we needed to match the generic interface due to how we handle triggers, it could be renamed to "notused" or something but i figured a name to represent what it actually is was more fitting

Reply:
It's part of that method signature, since it's used for every trigger